### PR TITLE
## - Bug fix on login and expired product

### PR DESCRIPTION
**Pull request by UwaweTera**
### What does this PR do?

Bug fix for login and expired product 

### Description of Task to be completed?

this branch fixing the login error message adding other functionality to exprired product route

### How should this be manually tested?

npm install
npm start

### Any background context you want to provide?

none

### What are the relevant pivotal tracker stories?

Provide a relevant link to the pivotal tracker.

### Before
before
After
after1

### Questions:

# Type of change

Please check the options that are relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or Enhancement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code is formatted correctly
- [x] My changes doesn't have any errors or warnings
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by Ntarecedrick — APPROVED** [review]

**Review by teerenzo — APPROVED** [review]

File: src/services/productService.js [hunk 1]
```diff
@@ -710,22 +713,20 @@ class ProductServices {
...
+    //   include: [
+    //     {
+    //       model: User,
+    //       as: "seller",
+    //     },
+    //   ],
+    // });
 
+    // const { name, email } = product.seller;
+    // await new SendEmail({ name, email }, null, product.name).expiredProduct();
```
Comment (jimmymut) [inline src/services/productService.js]: Do we need these comments?

Comment (jimmymut) [inline src/services/productService.js:728]: Do we need these comments?

Comment (jimmymut) [inline src/services/productService.js:728]: Do we need these comments?

Comment (jimmymut) [inline src/services/productService.js:728]: Do we need these comments?

**Comment by jimmymut** [comment]
I think this isProductExpired middleware can be added in adding "aproduct to cart" and "marking the product as available" endpoints. you can talk to the respective people who worked on those endpoints and come up with the solution may be

**Review by NG-Solennel — APPROVED** [review]

**Review by Eli250 — APPROVED** [review]

**Pull request merged**